### ISSUE TITLE
Don't open the output panel automatically

### ISSUE THREAD

**Issue by wehrstedt** [otris/vscode-janus-debug#142]
I disabled the auto connect for the server console. These are my settings:
```
"vscode-janus-debug.serverConsole": {
     "autoConnect": false
}
```

The console doesn't connect. So far so good, but the output panel will still be opened automatically when you start vscode.

**Comment by bkircher** [comment]
Are you sure this isn’t just VSCode’s default behavior?

**Comment by wehrstedt** [comment]
Jap, if you deactivate the extension you will see that the output panel wont show up

**Comment by bkircher** [comment]
Cool, thanks for trying out. As always, Pull Requests welcome 😉

**Comment by wehrstedt** [comment]
If I will have some time left, sure. But currently it's pretty rare... 🙈

**Comment by wehrstedt** [comment]
fixed in #143
